esque/cli/commands.py

```diff
@@ -629,7 +629,7 @@ def get_topics(state: State, prefix: str, output_format: str):
     required=False,
 )
 @click.option(
-    "-n", "--numbers", metavar="<n>", help="Number of messages.", type=click.INT, default=sys.maxsize, required=False
+    "-n", "--number", metavar="<n>", help="Number of messages.", type=click.INT, default=sys.maxsize, required=False
```

Comment:
This is a breaking change. I think we should keep `--numbers` around for compatibility reasons and emit a deprecation warning.

Reply:
So we agreed to just release a new version with breaking changes, right? CC: @hfjn

Reply:
if you bump the version number accordingly go for it.